dashboard/test/models/pd/workshop_test.rb

```diff
@@ -179,11 +179,11 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     create :pd_workshop, sessions: [session_on_day(11)]
     create :pd_workshop, sessions: [session_on_day_late(11)]
 
-    start_expected = [workshop_in_10_days_early, workshop_in_10_days, workshop_in_10_days_late].map(&:id)
-    assert_equal start_expected, Pd::Workshop.scheduled_start_in_days(10).all.map(&:id)
+    start_expected = [workshop_in_10_days_early, workshop_in_10_days, workshop_in_10_days_late].pluck(:id)
```

Comment:
array: no improvement

There are a few more like this below too. I'm ok with using `pluck` everywhere now that it's supported on Enumerable, but we should realize it's not a perf improvement in these cases.